.changeset/puny-yaks-retire.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fixes a regression introduced to fix an issue regarding orphan CSS styles.
```

Comment:
Maybe this is clearer about the user-facing impact?

```suggestion
Fixes a regression introduced in Astro v5.14.7 that caused `?url` imports to not work correctly. This release reverts [#14142](https://github.com/withastro/astro/pull/14142).
```

Reply:
That looks good, Chris! 👍